api/app/controllers/mno_enterprise/config_controller.rb

```diff
@@ -46,5 +47,9 @@ def available_locales
         }
       end
     end
+
+    def set_default_locale
```

Comment:
Would it be better do to this in [TenantConfig](https://github.com/maestrano/mno-enterprise/blob/4.0/core/app/models/mno_enterprise/tenant_config.rb) where we are reloading the settings?

Reply:
Discussed on Slack. TenantConfig does not have the available locales list on app load and cannot set default_locale.